### DISCUSSION TITLE
Handling Files in the API

### DISCUSSION THREAD

**Discussion by Fabian1410** [Budibase/budibase#5052] [Help]
#4949 

How I can handle Files in the new API? I want to send some files to my Database from zapier ....

**Comment by melohagan** [comment]
Hi @Fabian1410 

It may be possible to adapt the script this helpful post to create a new row with an attachment: https://github.com/Budibase/budibase/discussions/4995#discussioncomment-2419868 

I had quick look myself, although unfortunately I couldn't get the form-data module to work on Zapier.
